scripts/bench_python_cpp_algos.py

```diff
@@ -15,6 +15,13 @@
 from katana.local import Graph, analytics
 
 # TODO(giorgi): This script needs to be tested in CI.
```

Comment:
indeed, it would be nice to have some small tests to check that refactorings like the one performed in this PR make no changes to the input/output expected format